internal/testutil/testdata/jobmapping_generator.go

```diff
@@ -0,0 +1,362 @@
+package main
```

Comment:
copied from worker integration tests. Will eventually remove the worker one in favor of this one